ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -707,6 +708,8 @@ ethernet_interfaces:
     pim:
       ipv4:
         sparse_mode: < true | false >
+    mac_security:
+      profiles: < profile >
```

Comment:
Since we can only have one profile per interface, shouldn't this be "profile" ? @titom73

Reply:
Updated.